controller/PciManager.php

```diff
@@ -253,6 +278,24 @@ protected function getRequestPciDataObject()
         return null;
     }
 
+
+    /**
+     * @throws PortableElementException
+     * @throws PortableElementVersionIncompatibilityException
+     * @throws \common_Exception
+     * @throws \common_exception_Error
+     */
+    public function unregister()
+    {
+        $pci = $this->getRequestPciDataObject();
+        $this->itemsScannerService->isPciUsedInItems(
+            $this->getRequestParameter('typeIdentifier'),
```

Comment:
`isPciUsedInItems()` expects a string as a first param, so it might be good to validate `$this->getRequestParameter('typeIdentifier')` before passing it to avoid unexpected behaviour